src/Network/Network/LoadBalancer/NewAzLoadBalancerBackendAddressConfig.cs

```diff
@@ -74,13 +74,20 @@ public partial class NewAzureLoadBalancerBackendAddress : NetworkBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string LoadBalancerFrontendIPConfigurationId { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The admin state associated with the Backend Address config",
+            ValueFromPipelineByPropertyName = true)]
+        public string AdminState { get; set; }
```

Comment:
Could you provide argument completer for `AdminState`?

Reply:
fixed